syft/pkg/language.go

```diff
@@ -20,6 +20,7 @@ const (
 	Go              Language = "go"
 	Rust            Language = "rust"
 	Dart            Language = "dart"
+	Dotnet          Language = "dotnet"
```

Comment:
Open question: do we prefer `.net` or `dotnet` for string representations of these values?

Reply:
hm, I think `.net` can be used for `.NET Framework` and `.NET Core`. `dotnet` always just means `.NET Core`, so I would prefer `dotnet`.

Reply:
sounds great -- thanks for the context 👍